cmd/buildah/config.go

```diff
@@ -39,6 +39,8 @@ type configResults struct {
 	label                  []string
 	onbuild                []string
 	os                     string
+	osversion              string
+	osfeature              []string
```

Comment:
too keep my ocd happy, could you flip the order of these, feature at line 42, version at 43 please?

Reply:
Changing it.